libnopegl/src/backends/gl/pipeline_gl.c

```diff
@@ -713,8 +676,6 @@ void ngli_pipeline_gl_draw(struct pipeline *s, int nb_vertices, int nb_instances
     else
         ngli_glDrawArrays(gl, gl_topology, 0, nb_vertices);
 
-    unbind_vertex_attribs(s, gl);
```

Comment:
How is the unbind related to this commit?

Reply:
`The unbind_vertex_attribs()` was only there for the codepath not supporting VAOs.

Reply:
Can you mention that in the commit description?

Reply:
Done.